lib/anemone/page.rb

```diff
@@ -47,6 +48,8 @@ def initialize(url, params = {})
       @response_time = params[:response_time]
       @body = params[:body]
       @error = params[:error]
+      @skip_no_follow = params[:skip_no_follow]
+      @follow_subdomain = params[:follow_subdomain]
```

Comment:
i found this a bit confusing.
1) the name makes it sound like we are passing the actual subdomain, not the domain we want at the top level.
2)  We already have a url, so seems like this should be true/false.  If they want more complex matching, there is focus crawl method for matching.  We can parse the domain from the passed in URL.